src/lib/gameLogic.re

```diff
@@ -50,11 +50,12 @@ let transformBoard = (direction, board) => {
   |> orientBoard(direction, PostTransform)
 };
 
+let randomInt = (num) => Js.Math.random() *. float_of_int(num) |> Js.Math.floor_int;
+
 let addCell = (board) => {
-  Random.init([%bs.raw {|Math.random() * 10000|}]);
```

Comment:
You have access to all the JS APIs under Js.*: https://bucklescript.github.io/bucklescript/api/Js.html